pure_protobuf/protobuf.py

```diff
@@ -221,32 +257,44 @@ class Float32Type(Fixed32SubType):
 Bytes = BytesType()
 Unicode = UnicodeType()
 
-# Messages. --------------------------------------------------------------------
+# Messages. -------------------------------------------------------------------
+
 
 class Flags:
-    '''
+    """
     Flags for a field.
-    '''
+    """
+
+    SIMPLE = 0  # Single value field.
+    REQUIRED, REQUIRED_MASK = 1, 1  # Required field_type.
+    SINGLE, REPEATED, PACKED_REPEATED, REPEATED_MASK = (
+        0,
+        2,
+        6,
+        6,
+    )  # Repeated and packed-repeated fields.
+    PRIMITIVE, EMBEDDED, EMBEDDED_MASK = (
+        0,
+        8,
+        8,
+    )  # Used by MessageMetaType to see if a field has embedded definition.
 
-    SIMPLE = 0 # Single value field.
-    REQUIRED, REQUIRED_MASK = 1, 1 # Required field_type.
-    SINGLE, REPEATED, PACKED_REPEATED, REPEATED_MASK = 0, 2, 6, 6 # Repeated and packed-repeated fields.
-    PRIMITIVE, EMBEDDED, EMBEDDED_MASK = 0, 8, 8 # Used by MessageMetaType to determine if a field contains embedded definition.
 
 class EofWrapper:
-    '''
+    """
     Wraps a stream to raise EOFError instead of just returning of ''.
-    '''
+    """
+
     def __init__(self, fp, limit=None):
         self.__fp = fp
         self.__limit = limit
-        
+
     def read(self, size=None):
-        '''
+        """
         Reads a string. Raises EOFError on end of stream.
-        '''
+        """
         if size == 0:
-            return ''
+            return ""
```

Comment:
This should now depend on `fp` mode (binary or text). Unsure, if that breaks anything though.